crates/core/src/operations/mod.rs

```diff
@@ -283,6 +283,22 @@ pub fn get_num_idx_cols_and_stats_columns(
     )
 }
 
+/// Get the target_file_size from the table configuration in the sates
+/// If table_config does not exist (only can occur in the first write action) it takes
+/// the configuration that was passed to the writerBuilder.
+pub fn get_target_file_size(
```

Comment:
I don't think this should become part of our public API, so I'm going to slap a crate level visibility on it.

Reply:
Woops, my bad, should have been internal indeed